asab/storage/elasticsearch.py

```diff
@@ -50,27 +52,33 @@ def __init__(self, app, service_name, config_section_name='asab:storage'):
 		super().__init__(app, service_name)
 		self.Loop = app.Loop
 
-		self.URL = Config.get(config_section_name, 'elasticsearch_url')
-		parsed_url = urllib.parse.urlparse(self.URL)
-		self.ServerUrls = [
-			urllib.parse.urlunparse((parsed_url.scheme, netloc, parsed_url.path, None, None, None))
-			for netloc in parsed_url.netloc.split(',')
-		]
+		# new configuration format [elasticsearch]
+		url, username, password, api_key = 'url', 'username', 'password', 'api_key'
 
-		self.Refresh = Config.get(config_section_name, 'refresh')
-		self.ScrollTimeout = Config.get(config_section_name, 'scroll_timeout')
+		# old configuration format [asab:storage]
+		if config_section_name == 'asab:storage':
+			LogObsolete.warning(
+				"The configuration format [asab:stirage] is outdated. Please use an updated version [elasticsearch] with url, username and passowrd parameters."
+			)
+			url, username, password, api_key = 'elasticsearch_url', 'elasticsearch_username', 'elasticsearch_password', 'elasticsearch_api_key'
```

Comment:
Why you do this? ;-)

Reply:
Because I will have different sections for different configurations 
`[asab:storage]` with 'elasticsearch_url', 'elasticsearch_username', 'elasticsearch_password' 
or [`elastic]` with 'url', 'username', 'password', 'api_key'

Reply:
Can you do this explicitly then?

This unification represents a unnecessary cognitive load.

Reply:
In either case, please check the spelling ("stirage", "passowrd",...)